include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -600,6 +600,287 @@ struct SPhysicalDeviceLimits
+        if (maxTessellationControlPerVertexOutputComponents > _rhs.maxTessellationControlPerVertexOutputComponents) return false;
+        if (maxTessellationControlPerPatchOutputComponents > _rhs.maxTessellationControlPerPatchOutputComponents) return false;
+        if (maxTessellationControlTotalOutputComponents > _rhs.maxTessellationControlTotalOutputComponents) return false;
+        if (maxTessellationEvaluationInputComponents > _rhs.maxTessellationEvaluationInputComponents) return false;
+        if (maxTessellationEvaluationOutputComponents > _rhs.maxTessellationEvaluationOutputComponents) return false;
+        if (maxGeometryShaderInvocations > _rhs.maxGeometryShaderInvocations) return false;
+        if (maxGeometryInputComponents > _rhs.maxGeometryInputComponents) return false;
+        if (maxGeometryOutputComponents > _rhs.maxGeometryOutputComponents) return false;
+        if (maxGeometryOutputVertices > _rhs.maxGeometryOutputVertices) return false;
+        if (maxGeometryTotalOutputComponents > _rhs.maxGeometryTotalOutputComponents) return false;
+        if (maxFragmentInputComponents > _rhs.maxFragmentInputComponents) return false;
+        if (maxFragmentOutputAttachments > _rhs.maxFragmentOutputAttachments) return false;
+        if (maxFragmentDualSrcAttachments > _rhs.maxFragmentDualSrcAttachments) return false;
+        if (maxFragmentCombinedOutputResources > _rhs.maxFragmentCombinedOutputResources) return false;
+        if (maxComputeSharedMemorySize > _rhs.maxComputeSharedMemorySize) return false;
+        if (maxComputeWorkGroupCount[0] > _rhs.maxComputeWorkGroupCount[0]) return false;
+        if (maxComputeWorkGroupCount[1] > _rhs.maxComputeWorkGroupCount[1]) return false;
+        if (maxComputeWorkGroupCount[2] > _rhs.maxComputeWorkGroupCount[2]) return false;
+        if (maxComputeWorkGroupInvocations > _rhs.maxComputeWorkGroupInvocations) return false;
+        if (maxWorkgroupSize[0] > _rhs.maxWorkgroupSize[0]) return false;
+        if (maxWorkgroupSize[1] > _rhs.maxWorkgroupSize[1]) return false;
+        if (maxWorkgroupSize[2] > _rhs.maxWorkgroupSize[2]) return false;
+        if (subPixelPrecisionBits > _rhs.subPixelPrecisionBits) return false;
+        if (maxDrawIndirectCount > _rhs.maxDrawIndirectCount) return false;
+        if (maxSamplerLodBias > _rhs.maxSamplerLodBias) return false;
+        if (maxSamplerAnisotropyLog2 > _rhs.maxSamplerAnisotropyLog2) return false;
+        if (maxViewports > _rhs.maxViewports) return false;
+        if (maxViewportDims[0] > _rhs.maxViewportDims[0]) return false;
+        if (maxViewportDims[1] > _rhs.maxViewportDims[1]) return false;
+        if (viewportBoundsRange[0] > _rhs.viewportBoundsRange[0]) return false;
+        if (viewportBoundsRange[1] > _rhs.viewportBoundsRange[1]) return false;
+        if (viewportSubPixelBits > _rhs.viewportSubPixelBits) return false;
+        if (minMemoryMapAlignment > _rhs.minMemoryMapAlignment) return false;
```

Comment:
@devshgraphicsprogramming @deprilula28, I think the `min` limits needs the comparison the other way around.
I mean it's better to have smaller alignment requirements, right? :D

Reply:
yes a device with smaller alignment can fullfill the role of a device with a larger alignment